### PR TITLE
Update toolkit to v4

### PR DESCRIPTION
**Pull request by Birden**

### CODE REVIEW

File: src/app/components/crosssection/crosssection.component.ts [hunk 1]
```diff
@@ -193,7 +195,7 @@ export class CrosssectionComponent implements AfterViewInit {
               'textstyle': {
                 'color': '#757575'
               },
-              'anchor': AnchorType.LeftCenter
+              'alignment': AnchorType.LeftCenter
```
Comment (serrique) [inline src/app/components/crosssection/crosssection.component.ts:197]: I'd delete it. anchor is not alignment

Comment (Birden) [reply src/app/components/crosssection/crosssection.component.ts:197]: but then the axis labels will not be displayed

Comment (serrique) [reply src/app/components/crosssection/crosssection.component.ts:197]: okay

File: src/app/components/crosssection/crosssection.component.ts [hunk 2]
```diff
@@ -315,13 +317,10 @@ export class CrosssectionComponent implements AfterViewInit {
     track.addChild(image);
   }
   private addPanelHeader(widget: MultiWellWidget, track: CorrelationTrack, start: number, end: number) {
-    const headerGroup = widget.getTrackHeader(track);
+    const headerGroup = widget.getTrackHeader(track) as Group;
```
Comment (buryndin) [inline src/app/components/crosssection/crosssection.component.ts]: try to fix in toolkit

Comment (Birden) [reply src/app/components/crosssection/crosssection.component.ts]: I could't set the exact type for this method in toolkit, seems there are too more dependencies

File: package.json [hunk 3]
```diff
@@ -12,17 +12,17 @@
...
     "@angular/compiler": "^15.2.9",
     "@angular/core": "^15.2.9",
     "@angular/forms": "^15.2.9",
+    "@angular/material": "^15.2.9",
     "@angular/platform-browser": "^15.2.9",
     "@angular/platform-browser-dynamic": "^15.2.9",
     "@angular/router": "^15.2.9",
-    "@int/geotoolkit": "~3.4.119",
-    "@int/impl": "~3.4.119",
+    "@int/impl": "../../geotoolkit/bin/dist/int-impl-4.0.0.tgz",
```
Comment (buryndin) [inline package.json]: should be npm link

**Pull request merged**